### ISSUE TITLE
osc server crash...

### ISSUE THREAD

**Issue by ffd8** [ffd8/P5LIVE#20]
node.js server crashes at odd times (usually on page refresh) when using the develop branch w/ OSC implementation. pretty sure it has to do with osc server/port not being properly disconnected.. but maybe it's an issue with the node-osc module?

- [discussion](https://github.com/automata/osc-web/pull/8) from [osc-web](https://github.com/automata/osc-web) about issue that was somewhat resolved
- [discussion](https://github.com/nodejs/node/issues/7061) about possible issue within nodejs
- [possible fix](https://stackoverflow.com/questions/56003679/check-if-udp-socket-is-runing-on-a-certain-port-close-it-then-run-it-again/56016615)

error message: 
```
dgram.js:827
    throw new ERR_SOCKET_DGRAM_NOT_RUNNING();
    ^

Error [ERR_SOCKET_DGRAM_NOT_RUNNING]: Not running
....
```

**Comment by ffd8** [comment]
Simple fix.. [update node-osc package](https://github.com/ffd8/p5js-osc/commit/4886325f4a8a3750888ccd563b6ad78e137cd376)!

**Comment by ffd8** [comment]
Just kidding... updating package helped reduce it – but still occurs at random times (especially on page refresh)... any ideas out there??

**Comment by MylesBorins** [comment]
@ffd8 I've pushed an update to node-osc which should fix the new issue you were having with experimental module features... I think updates between 2.x -> 4.x should have fixed the disconnect error, but if you are still seeing it and can reproduce please let me know and I'll see how we can fix it!

**Comment by ffd8** [comment]
@MylesBorins Thanks for chiming in and the updates – unfortunately, [still an issue](https://github.com/MylesBorins/node-osc/issues/26)... Really curious how/why this occurs. There must be something additional I need to do to properly `close()` or whatnot before a reconnection occurs?

**Comment by ffd8** [comment]
*Solved* (hopefully) with #39 and continuing conversation on [more detailed thread](https://github.com/MylesBorins/node-osc/issues/26).
